README.md

```diff
@@ -34,9 +36,41 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+```sh
+./scripts/quick-helm-install.sh 
+```
```

Comment:
We're back at requiring user to `git clone` this repository. The whole point of Helm-based installation was to **not** require the user to clone this repo. Any ideas how to avoid that?

Reply:
Good point, perhaps we could do something like ´curl -0 https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh´? I think that should work.

Reply:
You can have a one-liner by piping curl output to `bash`. That's typically how it's done in other projects.
Please make sure curl will fail promptly on unsuccessful HTTP response status.

Reply:
I'll use `curl --fail-with-body https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh | bash` then

Reply:
Note that `--fail-with-body` may not be supported in older versions of `curl`. Curl 7.68.0 I have on Ubuntu 20.04 doesn't seem to know that. Just `--fail` should be more understood.